modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
+[id="logging-loki-reliability-hardening_{context}"]
+= Configuring Loki to tolerate node failure
+
+In logging 5.8 and newer versions, the Loki Operator supports setting pod anti-affinity rules to request that pods of the same component are scheduled on different available nodes in the cluster.
```

Comment:
ACK, I will add a Jira to follow up on making sure this is standard (I didn't realize it was "later" until very recently 😅).
RE the naming, Logging isn't an official product name AFAIK. We actually have a separate Jira to figure that out, but I think we have an attribute. It's not really correct, and very wordy ("the logging subsystem for OpenShift",  I think 😬) so I was trying to avoid using it since we'll need to replace / reword around it later, but for now I think it's the best way forward.